Editor/EditorCore/EditorHandleUtility.cs

```diff
@@ -511,24 +511,35 @@ internal static void PopMatrix()
         /// <param name="mousePosition"></param>
         /// <returns></returns>
         internal static Plane FindBestPlane(Vector2 mousePosition)
+        {
+            var res = FindBestPlaneAndBitangent(mousePosition);
+            return res.item1;
+        }
+
+        internal static SimpleTuple<Plane, Vector3> FindBestPlaneAndBitangent(Vector2 mousePosition)
         {
             // Priority in finding the "best" plane for input from a mouse position:
             // 1. Take the plane from the first hit mesh.
             // 2. If ProGrids is drawing a grid, use the plane normal and raycast for position
             // 3. Use the nearest matching plane based on the scene camera direction
 
             Plane plane;
+            Vector3 bitangent;
+
+            if (GetPlaneFromPickedObject(mousePosition, out plane, out bitangent))
+                return new SimpleTuple<Plane, Vector3>(plane, bitangent);
+
+            if (!GetPlaneFromProGridsAxis(mousePosition, out plane))
+                plane = GetPlaneFromCameraDirection();
 
-            if (GetPlaneFromPickedObject(mousePosition, out plane))
-                return plane;
+            var nrm = plane.normal;
 
-            if(GetPlaneFromProGridsAxis(mousePosition, out plane))
-                return plane;
+            bitangent = Vector3.Cross(nrm, Vector3.Cross(nrm, nrm == Vector3.right ? Vector3.forward : Vector3.right));
 
-            return GetPlaneFromCameraDirection();
+            return new SimpleTuple<Plane, Vector3>(plane, bitangent);
```

Comment:
Looks like `GetPlaneFromCameraDirection` is omitted from the process here. Was that intentional?